packages/clerk-js/src/core/auth/SessionCookieService.ts

```diff
@@ -0,0 +1,134 @@
+import { setDevBrowserJWTInURL } from '@clerk/shared/devBrowser';
+import { is4xxError, isClerkAPIResponseError, isNetworkError } from '@clerk/shared/error';
+import type { Clerk, EnvironmentResource } from '@clerk/types';
+
+import { clerkCoreErrorTokenRefreshFailed, clerkMissingDevBrowserJwt } from '../errors';
+import { eventBus, events } from '../events';
+import type { FapiClient } from '../fapiClient';
+import type { ClientUatCookieHandler } from './cookies/clientUat';
+import { createClientUatCookie } from './cookies/clientUat';
+import type { SessionCookieHandler } from './cookies/session';
+import { createSessionCookie } from './cookies/session';
+import type { DevBrowser } from './devBrowser';
+import { createDevBrowser } from './devBrowser';
+import { SessionCookiePoller } from './SessionCookiePoller';
+
+// TODO: make SessionCookieService singleton since it handles updating cookies using a poller
+// and we need to avoid updating them concurrently.
+export class SessionCookieService {
```

Comment:
It's hard to see the diff of this file as it was moved...can you outline the changes made here briefly?

Reply:
We probably need to rename the service to `AuthCookieService`. My intention is to create an interface that can be used both for Browser / Native apps and use 2 different implementations. The `SessionCookieService` will be used for Browser.

Changes:
- move code cookies related with auth to this service
- introduce auth related helper methods that use auth cookies to avoid exposing cookies handling outside of this module
- apply changes due to refactoring cookies exposed interface